posthog/temporal/data_modeling/run_workflow.py

```diff
@@ -272,35 +273,53 @@ async def handle_model_ready(model: ModelNode, team_id: int, queue: asyncio.Queu
         team_id: The ID of the team who owns this model.
         queue: The execution queue where we will report back results.
     """
+
+    async def handle_error(error: Exception, error_message: str):
+        await logger.aexception(error_message, model.label, str(error))
+
+        if "job_id" in locals():
```

Comment:
Anyway we could get `job_id` as a parameter here? Perhaps as an attribute of the `ModelNode`? I may be nit picking but I find it very hard to understand what's the expectation when using `locals`.

Reply:
Yes, this is hard for me to understand as well. I think we can do better in terms of readability here and I'll add a change for this.